drivers/soundwire/stream.c

```diff
@@ -698,7 +698,7 @@ static int sdw_bank_switch(struct sdw_bus *bus, int m_rt_count)
 	else
 		ret = sdw_transfer(bus, wr_msg);
 
-	if (ret < 0) {
+	if (ret < 0 && ret != -ENODATA) {
```

Comment:
typo in the commit message "This changed"

Reply:
fixed.